src/js/node/http2.ts

```diff
@@ -2041,14 +2043,14 @@ class ServerHttp2Stream extends Http2Stream {
   }
   respondWithFD(fd, headers, options) {
     if (this.destroyed || this.closed) {
-      throw $ERR_HTTP2_INVALID_STREAM(`The stream has been destroyed`);
+      throw $ERR_HTTP2_INVALID_STREAM();
     }
-    if (this.headersSent) throw $ERR_HTTP2_HEADERS_SENT("Response has already been initiated.");
+    if (this.headersSent) throw $ERR_HTTP2_HEADERS_SENT();
 
     if (headers == undefined) {
       headers = {};
     } else if (!$isObject(headers)) {
-      throw $ERR_HTTP2_INVALID_HEADERS("headers must be an object");
+      throw $ERR_INVALID_ARG_TYPE("headers", "object", headers);
```

Comment:
How did you find this difference?

Reply:
after removing the autogenerated error code type definitions i went and audited all the uses, and realized `ERR_HTTP2_INVALID_HEADERS` doesn't actually exist and replaced it with the proper one instead.